### PR TITLE
Issue #1609 - Improve Kustomize 2 parameters UI

### PR DESCRIPTION
**Pull request by alexmt**

### CODE REVIEW

File: src/app/applications/components/application-parameters/kustomize.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+export interface Image {
+    name: string;
+    newName?: string;
+    newTag?: string;
+    digest?: string;
+}
+
+const pattern = /^(.*):([a-zA-Z0-9._-]*)$/;
+
+function parseOverwrite(arg: string, overwriteImage: boolean): { name: string; digest?: string, tag?: string } {
```
Comment (alexec) [inline src/app/applications/components/application-parameters/kustomize.tsx]: This is BL in the FE. Perhaps it's time for `yarn test`?

**Review by alexec — APPROVED** [review]
I'm going to give a rubber-stamp approval for this PR. Blast area is small and I trust you to have testing it adequately.

File: src/app/applications/components/application-parameters/kustomize.tsx (hunk 1 above)
Comment (alexmt) [reply src/app/applications/components/application-parameters/kustomize.tsx]: I agree. Lets add first unit tests. Working on it

Comment (alexmt) [reply src/app/applications/components/application-parameters/kustomize.tsx]: done: https://github.com/argoproj/argo-cd-ui/pull/125/files#diff-513b5bbde593665bce8f6bf307851b8c

**Pull request merged**